# HG changeset patch
# User Rik <rik@octave.org>
# Date 1512776764 28800
#      Fri Dec 08 15:46:04 2017 -0800
# Node ID 583e6447d7dc5274c5fe8e20bba32bf2126ba3fa
# Parent  06a7646a5619de6589452c318c3deed43c1f5ecb
camzoom.m: Add a tolerance of 2*eps to failing BIST test or Mac OS X.

* camzoom.m: Relax tolerance from 0, to 2*eps.

diff --git a/scripts/plot/appearance/camzoom.m b/scripts/plot/appearance/camzoom.m
--- a/scripts/plot/appearance/camzoom.m
+++ b/scripts/plot/appearance/camzoom.m
@@ -138,17 +138,17 @@ endfunction
 %! unwind_protect
 %!   hax1 = subplot (1, 2, 1);
 %!   sphere (hax1);
 %!   hax2 = subplot (1, 2, 2);
 %!   sphere (hax2);
 %!   camzoom (hax1, 2)
 %!   x = camva (hax1);
 %!   y = camva (hax2);
-%!   assert (tand (y/2) / tand (x/2), 2)
+%!   assert (tand (y/2) / tand (x/2), 2, 2*eps)
 %! unwind_protect_cleanup
 %!   close (hf);
 %! end_unwind_protect
 
 ## Test input validation
 %!error <Invalid call> camzoom (1, 2, 3)
 %!error <Invalid call> camzoom (1, [2, 3])
 %!error <Invalid call> camzoom ("string")
